docs/src/docs/usage/faq.mdx

```diff
@@ -32,3 +32,22 @@ The same as the Go team (the 2 last minor versions)
 
 Because the first run caches type information. All subsequent runs will be fast.
 Usually this options is used during development on local machine and compilation was already performed.
+
+## Why do you have `typecheck` errors?
+
+`typecheck` is like the front-end of a Go compiler, parses and type-checks Go code, it manages compilation errors.
+
+It cannot be disabled because of that.
+
+Of course, this is just as good as the compiler itself and a lot of compilation issues will not properly show where in the code your error lies.
+
+`typecheck` is not a real linter, it's just a way to parse compiling errors (produced by the `types.Checker`) and some linter errors.
+
+A linter that reports `typecheck` errors is not able to report other issues for a package because that kind of error doesn't allow it to perform its own analysis.
```

Comment:
```suggestion
A result that reports `typecheck` errors is not able to report other issues for a package because that kind of error doesn't allow it to perform its own analysis.
```

Reply:
In this context, it's the case where a linter (not typecheck) reports a typecheck errors.
It's related to the previous sentence "and some linter errors".

IMO, we can keep the sentence or the sentence must rewritten because the word "result" cannot be a substitute of "linter" in the context of the sentence.

Reply:
I rewrote the sentence.

Reply:
Amazing, very clear!